packages/itwinui-react/src/core/Tabs/Tabs.tsx

```diff
@@ -177,78 +85,147 @@ export const Tabs = (props: TabsProps) => {
   }
 
   const {
-    labels,
-    activeIndex,
-    onTabSelected,
-    focusActivationMode = 'auto',
-    type = 'default',
-    color = 'blue',
+    className,
+    children,
     orientation = 'horizontal',
-    tabsClassName,
-    contentClassName,
-    wrapperClassName,
+    type = 'default',
+    ...rest
+  } = props;
+
+  const [currentActiveIndex, setCurrentActiveIndex] = React.useState(0);
```

Comment:
Maybe we could add a prop for users to choose initial active index? In case they want to show something else than Tab0.

Reply:
[This comment](https://github.com/iTwin/iTwinUI/pull/1401#discussion_r1275298092) asked for the removal of these two props. Instead, if users want to set an initial active index, they can pass the `isActive` prop to the tab which essentially nullifies the use of `currentActiveIndex`. Both of the overflow examples in Storybook are examples of this method in action

Reply:
i think we should do away with the concept of indices. users want to select the tab, not the index.

i expect controlled usage to look like this:
```jsx
const [current, setCurrent] = React.useState('apple');

<Tabs>
  <Tabs.TabList>
    {['apple', 'banana', 'orange'].map(label => (
      <Tabs.Tab
       key={label}
       label={label}
       isActive={current === label}
       onActiveChange={() => setCurrent(label)}
      />
    ))}
  </Tabs.TabList>
</Tabs>
```

Reply:
This is a much cleaner usage! I've changed the stories and website to reflect this